brainstormforce/ultimate-addons-for-gutenberg
# Fix reusable forms checkbox radio button issue on front-end

[Trello](https://trello.com/c/znk20jJa/224-package-testing-1220)

### Description
- Fixed Radio Button, Toggle, Accept, and Checkbox block issues on the front-end when we add these forms. 

#### Test Cases
- Add form on the editor 
- Add radio button, checkbox, toggle, and accept field.
- Add that form to reusable block
- Add a new form on the editor
- Add radio button, checkbox, toggle, and accept field.
- check the front-end all the fields and stylings are working properly.

### Types of changes
- Added reusable block id and added CSS to forms block.

### How has this been tested?
Checked whether it is working on front-end or not.

### Checklist:
